src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String welcome;
```

Comment:
What is `welcome`?

Reply:
Welcome is the same as hello or greeting.